app/views/macros/claim-confirmation-form.njk

```diff
@@ -0,0 +1,12 @@
+{% macro viewClaimConfirmationForm(applicationId, page, crumb) %}
+  <div class="govuk-panel govuk-panel--confirmation govuk-!-text-align-left">
+    <h1 class="govuk-panel__title-s govuk-!-font-size-36 govuk-!-margin-top-1">Are you sure you want to process claim?</h1>
```

Comment:
This content doesn't tie in with [User Story 148784](https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/148784)